tests/test_log_scanner.py

```diff
@@ -638,8 +638,27 @@ def test_otj_premier_p1p1_ocr_multiclick(mock_screenshot, mock_ocr, otj_scanner)
     assert expected_names == card_names, f"OCR Test Failed: OCR Pack Cards, Set: OTJ, Expected: {expected_names}, Actual: {card_names}"
 
     # Simulate refresh clicks
-    otj_scanner.draft_data_search(Source.REFRESH)
-    otj_scanner.draft_data_search(Source.REFRESH)
+    otj_scanner.draft_data_search(True, False)
+    otj_scanner.draft_data_search(True, False)
 
     # Verify that the OCR method was only called once
-    assert mock_ocr.call_count == 1
\ No newline at end of file
+    assert mock_ocr.call_count == 1
+
+@patch("src.log_scanner.OCR.get_pack")
+@patch("src.log_scanner.capture_screen_base64str")
+def test_otj_premier_p1p1_ocr_disabled(mock_ocr, otj_scanner):
```

Comment:
You're missing the `mock_screenshot` argument. 
This should read `def test_otj_premier_p1p1_ocr_disabled(mock_screenshot, mock_ocr, otj_scanner):`